articles/user-guide/basics.md

```diff
@@ -10,105 +10,104 @@ uid: microsoft.quantum.guide.basics
 
 ### Types
 Q# is a strongly-typed language, such that careful use of types can help the compiler provide strong guarantees about Q# programs at compile time.
-In addition to standard and quantum-specific built-in primitive types (e.g. `Int`, `Bool`, `Qubit`, and `Result`), Q# provides support for user-defined types.
-All of Q#'s various primitive types are described on the [Types in Q#](xref:microsoft.quantum.guide.types) page, along with details on array and tuple types, as well as how to define new types within a Q# file.
+In addition to standard and quantum-specific built-in primitive types, for example, `Int`, `Bool`, `Qubit`, and `Result`, Q# provides support for user-defined types.
+
+For descriptions of all the primitive types, details for array and tuple types, and steps to define new types within a Q# file, see [Types in Q#](xref:microsoft.quantum.guide.types).
 
 ### Expressions
 An expression in a programming language is a combination of one or more constants, variables, operators, and functions that the programming language interprets and evaluates to a specific value.
 Most simply, for every type in a language, expressions of that type can be either *literals* or symbols bound to a value of that type.
 For example, `5` is an `Int` literal (thus also an expression of type `Int`), and if the symbol `count` is bound to the integer value `5`, then `count` is also an integer expression.
 
-Additionally, an expression can consist of other expressions combined with certain operators.
-Hence another example of an `Int` expression which evaluates to `5` is `2+3`.
+Additionally, an expression can consist of other expressions combined by certain operators.
+For example, another example of an `Int` expression that evaluates to `5` is `2+3`.
 
-The possible expressions of types in Q#, as well as the compatible operators that can be used to form them, are detailed on the [Type Expressions in Q#](xref:microsoft.quantum.guide.expressions) page. 
+For more information about expressions and compatible operators in Q#, see [Type Expressions in Q#](xref:microsoft.quantum.guide.expressions). 
 
 ### Statements 
-A statement is a syntactic unit of an imperative programming language that expresses some action to be carried out.
-Statements contrast with expressions in that statements do not return results and are executed solely for their side effects, while expressions always return a result and often do not have side effects at all.
-This distinction is frequently observed in wording: an expression is evaluated, whereas a statement is executed.
+A statement is a syntactic unit of an imperative programming language that expresses some action to carry out.
+Statements contrast with expressions in that statements do not return results and are executed solely for their side effects. Expressions, however, always return a result and often do not have side effects at all. In short, the Q# compiler executes a statement and evaluates an expression.
```

Comment:
Agreed it's the right gist, but I wouldn't think that the Q# compiler executes any statements at all; that would be up to the runtime or even more generally, to the chosen execution target. I'd suggest pinging @bettinaheim for additional input here.

Reply:
I think re-phrasing this in passive voice would be clearer, since for this topic it's not as important who/what is doing the execution - "In short, Q# statements are executed, while expressions are evaluated."